src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -61,8 +90,8 @@ const useStyles = makeStyles({
     },
   },
   sectionHeaderText: {
-    fontSize: '1.5em !important',
     fontWeight: '600 !important',
+    fontSize: '1.1em!important',
```

Comment:
Why remove the space before `!important`? I don't know if it changes things, but all the documentation I can see has a space there